### PR TITLE
1 add pages and routing

### LINKED ISSUE #1: Add pages and routing
- Add pages
- Add basic routing

### PR DESCRIPTION
**Pull request by rdshinde**
- Added pages
- Added basic routing
chrome-capture-2022-3-1

### CODE REVIEW

File: src/components/sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -9,39 +9,42 @@ export const Sidebar = () => {
...
               <div className="text-start text-4 bold-lg nav-link">
                 <span className="m-r-md">
                   <i className="fa fa-heart"></i>
                 </span>
                 Liked Videos
               </div>
             </NavLink>
-            <NavLink to={"#"} style={{ textDecoration: "none" }}>
+            <NavLink to={"/watch-later"} style={{ textDecoration: "none" }}>
               <div className="text-start text-4 bold-lg nav-link">
```
Comment (raman1999) [inline src/components/sidebar/Sidebar.jsx:37]: why are u using inline styles , can give class link for resetting properties

File: src/components/video-card/VideoCard.jsx [hunk 2]
```diff
@@ -29,10 +29,9 @@ export const VideoCard = () => {
   };
 
   return (
-    <div className="video-card-wrapper">
+    <div className="video-card-wrapper" style={{ maxHeight: "22rem" }}>
       <Video data={{ setActionModalState }} />
```
Comment (raman1999) [inline src/components/video-card/VideoCard.jsx:33]: here also inline

File: src/components/liked-video/LikedVideo.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+import { ActionModal } from "../action-modal/ActionModal";
+import { Video } from "../utility-video-card/Video";
+export const LikedVideo = () => {
+  const [showActionModal, setActionModalState] = useState(false);
+  const actionModalHandler = (type) => {
+    setActionModalState((prev) => !prev);
+    console.log(type);
+  };
```
Comment (raman1999) [inline src/components/liked-video/LikedVideo.jsx:8]: remove console

File: src/components/watch-later-video/WatchLaterVideo.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+import { ActionModal } from "../action-modal/ActionModal";
+import { Video } from "../utility-video-card/Video";
+export const WatchLaterVideo = () => {
+  const [showActionModal, setActionModalState] = useState(false);
+  const actionModalHandler = (type) => {
+    setActionModalState((prev) => !prev);
+    console.log(type);
+  };
```
Comment (raman1999) [inline src/components/watch-later-video/WatchLaterVideo.jsx:8]: remove console

File: src/pages/liked-video-page/LikedVideoPage.jsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+import { LikedVideo } from "../../components/liked-video/LikedVideo";
+export const LikedVideoPage = () => {
+  return (
+    <div>
+      <h3 className="playlist__heading">Your Liked Videos</h3>
+      <div className="video-container-wrapper" >
+        <LikedVideo />
+        <LikedVideo />
+        <LikedVideo />
+        <LikedVideo />
```
Comment (raman1999) [inline src/pages/liked-video-page/LikedVideoPage.jsx:11]: do change this later to mapping

**Review by raman1999 — COMMENTED** [review]
looks good, suggested some changes

File: src/components/liked-video/LikedVideo.jsx (hunk 3 above)
Comment (rdshinde) [reply src/components/liked-video/LikedVideo.jsx:8]: Okay

File: src/pages/liked-video-page/LikedVideoPage.jsx (hunk 5 above)
Comment (rdshinde) [reply src/pages/liked-video-page/LikedVideoPage.jsx:11]: Sure

**Review by raman1999 — APPROVED** [review]
Looks good now

File: src/components/video-card/VideoCard.jsx [hunk 6]
```diff
@@ -29,10 +29,9 @@ export const VideoCard = () => {
   };
 
   return (
-    <div className="video-card-wrapper">
+    <div className="video-card-wrapper" style={{ maxHeight: "22rem" }}>
```
Comment (Priyanshi-Tiwari) [inline src/components/video-card/VideoCard.jsx:32]: just remove the inline styling later other than that the setup looks good

**Pull request merged**